weekly-mission(week2)/src/features/signIn.js

```diff
@@ -1,20 +1,19 @@
-import MASTER_ACCOUNT from "../constants/accounts.js";
-import { $ } from "../constants/dom.js";
 import { addErrorMsgToIncorrectInput } from "../view/addErrorMessageToInput.js";
+import { emailInput, passwordInput } from "../constants/dom.js";
+import signIn from "../apis/signInUser.js";
 
-const isMaster = (email, password) => email === MASTER_ACCOUNT.EMAIL && password === MASTER_ACCOUNT.PASSWORD;
-
-export const signInHandler = (e) => {
+export const signInHandler = async (e) => {
   e.preventDefault();
 
-  const emailInput = $('.email').value;
-  const passwordInput = $('.password').value;
+  const inputAccount = {
```

Comment:
이정도로만 작성되어도 괜찮을 것 같아요!
```suggestion
  const account = {
```